pkg/microservice/aslan/core/environment/service/bundle.go

```diff
@@ -85,7 +85,7 @@ func GetBundleResources(logger *zap.SugaredLogger) ([]*resourceSpec, error) {
 				resourceSpec.Spec[v.Key] = v.Value
 			}
 		} else {
-			logger.Warnf("can not find resource key :%s", resourceKey)
+			//logger.Warnf("can not find resource key :%s", resourceKey)
```

Comment:
Why comment out this sentence?

Reply:
xunying asked me to comment on it,leave it later to improve

Reply:
Delete useless code, or add comments explaining subsequent behavior.

Reply:
add todo